media-libs/mlt/mlt-6.18.0.recipe

```diff
@@ -0,0 +1,69 @@
+SUMMARY="An open source multimedia framework"
+DESCRIPTION="A LGPL multimedia framework designed for TV broadcasting"
+HOMEPAGE="https://www.mltframework.org/"
+COPYRIGHT="2011-2019 by Meltytech, LLC."
+LICENSE="GNU LGPL v2.1"
+REVISION="1"
+SOURCE_URI="https://github.com/mltframework/mlt/releases/download/v$portVersion/mlt-$portVersion.tar.gz"
+CHECKSUM_SHA256="fd2a346e0cc093e5d813e04cd126c077a2d224ae4d17b5226ff7478feaa445ea"
+PATCHES="mlt-$portVersion.patchset"
+
+ARCHITECTURES="x86_64"
+SECONDARY_ARCHITECTURES="x86"
+
+libMltVersionCompat="$portVersion compat >= ${portVersion%%.*}"
+libMltPPVersionCompat="$portVersion compat >= 3"
```

Comment:
@Begasus I know why you suggest this, but I would suggest to dtop this libMltPPVersionCompat, because it is not needed. We can declare something that makes sense, without sticking to weird link numbering from upstream.

Reply:
@korli you are the master in these matters :) I'm following on what's presented after a full build, should we follow just "compat >= 6" as in the previous line then?

Reply:
How to declare?

Reply:
Yes, no need to overload ourselves. It will ease maintenance anyway. Just drop libMltPPVersionCompat and use libMltVersionCompat instead.

Reply:
OK, then @TURX can just use libVersion and libVersionCompat as default variables